src/message/MessageContainer.js

```diff
@@ -35,7 +30,7 @@ export default class MessageContainer extends React.PureComponent {
         twentyFourHourTime={twentyFourHourTime}
         selfEmail={auth.email}
         doNarrow={doNarrow}
-        onPress={this.handlePress}
+        onLongPress={() => handleLongPress(message)}
```

Comment:
Also, usually the property would be called onLongPress, while the handler inside the class would be handleLongPress.

So the function would be something like:

```js
handleLongPress = () => {
  const { message, onLongPress } = this.props;
  onLongPress(message);
}
```